lfs/batcher.go

```diff
@@ -11,16 +11,16 @@ import "sync/atomic"
 type Batcher struct {
 	exited     uint32
 	batchSize  int
-	input      chan Transferable
-	batchReady chan []Transferable
+	input      chan interface{}
```

Comment:
The question is why it uses a specific type when this code has no need of it. I was going to re-use the `Batcher` for something else and may still do (and will probably move it to another package), but didn't need to in the end. But what was clear is that the Batcher simple takes a stream of individual objects and turns them into a slice of N size - it has nothing to do with `Transferable`.